examples/acados/pendulum.py

```diff
@@ -32,7 +32,7 @@ def prepare_ocp(biorbd_model_path, final_time, number_shooting_points, nb_thread
     objective_functions.add(
         Objective.Mayer.MINIMIZE_STATE,
         weight=50000.0,
-        target=data_to_track.T,
+        target=data_to_track[-2:-1, :].T,
```

Comment:
This was a mistake of mine. This track the before last one point. To track the very last one, it should be `[-1:, :]`

Reply:
Yes, I noticed afterwards ;)